k8s/helm/values.yaml

```diff
@@ -0,0 +1,160 @@
+# Default values for arcadedb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+arcadedb:
+  # Enable persistence by updating this and volume/Mounts
+  databaseDirectory: "/home/arcadedb/databases"
+  defaultDatabases: "Universe[elon:musk]"
```

Comment:
may we use another dafaultDatabase without any reference to real people?

Reply:
Hehe, ya, only put this because it's in the example manifest https://github.com/ArcadeData/arcadedb/blob/2cbf66ea96f0f1d86c0512c557d94e55c9892590/package/src/main/config/arcadedb-statefulset.yaml#L111
And also exists in Java code so I thought there was some affinity. I'll change it though.

Reply:
i wasn't aware of this. I'll clean it too :)